<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlDataShowDialog</class>
 <widget class="QDialog" name="SqlDataShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SqlDataShowDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#SqlDataShowDialog{
border: 0px solid #000000;
background-color: #232323;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background-color:#232323;
border: 0px solid #000000;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>488</width>
        <height>365</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#scrollAreaWidgetContents{
 background-color: #232323;
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="testRecordView" name="tableView">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_recordcount">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgba(255, 255, 255, 0.7);
font-size: 14px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_page">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgba(255, 255, 255, 0.7);
font-size: 14px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_info">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
color: rgba(255, 255, 255, 0.7);
font-size: 14px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgba(255, 255, 255, 0.6);
font-size: 14px;</string>
           </property>
           <property name="text">
            <string>跳转到:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_page">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox{
background: #151515;
border-radius: 1px;
border: 1px solid #484848;
color: rgba(255, 255, 255, 0.8);
font-size: 16px;

}

QSpinBox::up-arrow{
    image:url(:/resources/icons/up.png);
border: 0px solid #484848;
}
QSpinBox::down-arrow{
    image:url(:/resources/icons/down.png);
border: 0px solid #484848;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>testRecordView</class>
   <extends>QTableView</extends>
   <header>testRecordView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sqlsettingdialog.qrc"/>
 </resources>
 <connections/>
</ui>
